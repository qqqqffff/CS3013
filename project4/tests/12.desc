eval FIFO analysis with 1 job
