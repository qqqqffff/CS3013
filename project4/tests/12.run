./scheduler FIFO tests/12.in 0
