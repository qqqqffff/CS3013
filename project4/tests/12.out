Begin analyzing FIFO:
Job 0 -- Response time: 0  Turnaround: 3  Wait: 0
Average -- Response: 0.00  Turnaround: 3.00  Wait: 0.00
End analyzing FIFO.
